src/oge/consumed.py

```diff
@@ -70,10 +55,16 @@
 ADJUSTMENTS = ["for_electricity", "for_electricity_adjusted"]
 
 
-def get_column(poll: str, adjustment: str, ba: str = ""):
-    """
-    Return output file column name for a poll and adjustment type
-    Returns mass columns, not rate columns
+def get_column(poll: str, adjustment: str, ba: str = "") -> str:
```

Comment:
should we rename this something like `get_pollutant_mass_column_name` for clarity?

Reply:
Done